### DISCUSSION TITLE
Option nodes

### DISCUSSION THREAD

**Discussion by Olatechworld** [srl-labs/containerlab#2216] [Q&A]
Hello everyone, 
Please how add option hostname and affect privileged for my node in my deployment file.
Thank you

**Comment by hellt** [comment]
Hi
what do you mean by `add option hostname`? The hostname will be set based on the node name in your yaml

**Reply by Olatechworld** [reply]
I mean i need to set the fqdn for my host and set privileged running for my container. Freeipa container need this options to running correctly.
Thank you
